spacy_llm/ty.py

```diff
@@ -131,6 +131,21 @@ def __getitem__(self, doc: Doc) -> Optional[Doc]:
         """
 
 
+@runtime_checkable
+class CandidateSelector(Protocol):
+    def __call__(self, mentions: Iterable[Span]) -> Iterable[Dict[str, str]]:
+        """Return list of Candidates with their descriptios for given mention and context.
```

Comment:
```suggestion
        """Return list of Candidates with their descriptions for given mention and context.
```

Reply:
Fixed in https://github.com/explosion/spacy-llm/pull/241/commits/0e7a28d74e30ddb6950933c0227cd92df9b4af94.